homeassistant/components/asuswrt/__init__.py

```diff
@@ -51,15 +55,15 @@
                     cv.ensure_list, [vol.In(SENSOR_TYPES)]
                 ),
                 vol.Optional(CONF_INTERFACE, default=DEFAULT_INTERFACE): cv.string,
-                vol.Optional(CONF_DNSMASQ, default=DEFAULT_DNSMASQ): cv.isdir,
+                vol.Optional(CONF_DNSMASQ, default=DEFAULT_DNSMASQ): cv.string,
```

Comment:
Wasn't this already changed in the other PR? Do we need to rebase here?

Reply:
I changed here because I was thinking that checks was failing for this, so I align code with the other PR.

Reply:
The other PR is merged so then this diff shouldn't exist. I think we need to rebase this PR on latest dev branch.

Reply:
Rebased